ovirt_imageio/_internal/auth.py

```diff
@@ -21,7 +21,10 @@ def __init__(self, ticket_dict, cfg):
             raise errors.InvalidTicket(
                 "Invalid ticket: %r, expecting a dict" % ticket_dict)
 
-        self._uuid = _required(ticket_dict, "uuid", str)
+        self._uuid = _optional(ticket_dict, "uuid", str)
+        if self._uuid is None:
+            # Require 'name' and leave 'uuid' as a legacy key.
+            self._uuid = _required(ticket_dict, "name", str)
```

Comment:
I would not keep the `_uuid` and `uuid` property - these are internal implementation
details and we can easy change them to `_name` and `name`. But maybe `_id` and `id` 
is a better term, since `name` does not have to be unique, but `id` is clearly is.

Regardless, changing clients to send the `name` instead of `uuid` is problematic
since vdsm is already assuming that a ticket has a `uuid`, and engine must support
old vdsm and imageio that require `uuid`. We can change engine to send both `name`
and `uuid` but I don't think such change is worth the effort.

We also use the uuid in the admin tool:

```
$ ./ovirt-imageioctl add-ticket -h
usage: ovirt-imageioctl add-ticket [-h] [-c CONF_DIR] filename

positional arguments:
  filename              Ticket filename.

options:
  -h, --help            show this help message and exit
  -c CONF_DIR, --conf-dir CONF_DIR
                        Configuration directory (default /etc/ovirt-imageio).
```

We can change the tool to require a ticket-id, but having all the details
in the actual ticket json is nice, so I would not do this chagne.

If we change the `uuid` internally, and change the examples to match the new internal
name, we will have tickets cannot be used with ovirt (e.g. manually adding the ticket to vdsm). This is not great situation.

So I think this work simply does not worth the effort, and it is better to keep the
current state.

The new tool can use the term ticket_id instead of uuid, for example:

    entrypoint --ticket-id xxx-yyy /path/to/image/or/socket